notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        compress_to_fp16 = False
</pre><p>if all parameters aligned probably there is no need in separation anymore?</p><p>BTW, does finally work on GPU?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>